npm_and_yarn/lib/dependabot/npm_and_yarn/helpers.rb

```diff
@@ -74,9 +70,21 @@ def self.run_yarn_commands(*commands)
             SharedHelpers.run_shell_command("yarn config set caFilePath #{ca_file_path}")
           end
         end
+      end
+      # Run any number of yarn commands while ensuring that `enableScripts` is
+      # set to false. Yarn commands should _not_ be ran outside of this helper
+      # to ensure that postinstall scripts are never executed, as they could
+      # contain malicious code.
+      def self.run_yarn_commands(*commands)
+        setup_yarn_berry
         commands.each { |cmd| SharedHelpers.run_shell_command(cmd) }
       end
 
+      def self.run_yarn_command(command)
```

Comment:
I think this is functionally the same as the `run_yarn_commands` method we already have, where we can pass one or more arguments, if we're worried about this reading weird, we could maybe alias the method, or delegate to the original method, to avoid the two from diverging in functionality over time?

Reply:
Ah, I see, yeah that makes sense. I wish the difference between these methods was a bit more evident but I'm not sure what we can reasonably change to make it so